.env

```diff
@@ -34,5 +34,7 @@ CONTENT_PREVIEW_MODE=true
 # Rollbar (only if you want to test errors are being reported locally)
 # ROLLBAR_ACCESS_TOKEN="..."
 # ROLLBAR_ENV=dev
+# VUE_APP=ROLLBAR_ACCESS_TOKEN="..."
+# VUE_APP_ROLLBAR_ENV=ROLLBAR_ENV
```

Comment:
Could just reuse `ROLLBAR_ENV` directly in the code, rather than declaring a new env var for this?

Reply:
Sadly not :( Vue only picks up env variables prefixed by VUE_APP. https://cli.vuejs.org/guide/mode-and-env.html#environment-variables

We could remove the ROLLBAR_ENV all together and just use VUE_APP_ROLLBAR_ENV but it didn't seem right to start referencing VUE_APP_ROLLBAR_ENV on the server.

Reply:
We shouldn't using the VueJS mechanism for env vars here – this won't work in production anyway (since in production we only have pre-built JS assets, and Vue components are instantiated via the server rendered HTML by the Rails app).

The best way to do this is similar to how we pass the Firebase config from the env to the client code, via regular data props (in the case of the Rollbar config, this should probably be done at the top level Vue app instantiation; for Firebase we are currently just passing this through to the SignIn and Profile components specifically).

Reply:
Done in: https://github.com/chaynHQ/soulmedicine/pull/163/commits/d4bdb5208503b139240d966b222f3cb2ace4f10b